docs/developers/adscertsigner.md

```diff
@@ -0,0 +1,90 @@
+##Ads Cert
```

Comment:
I recommend moving this up to just /docs instead of /docs/developers. Make it easier to discover. :)

Reply:
Moved